src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,23 @@
+package seedu.addressbook.commands;
+
+/**
+ * Sorts the address book by alphabetical order.
+ */
+public class SortCommand extends Command {
+
+    public static final String COMMAND_WORD = "sort";
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Sorts the address book by alphabetical order.\n"
+            + "Example: " + COMMAND_WORD;
+
+    public static final String MESSAGE_SUCCESS = "Address book has been sorted!";
+
+    public SortCommand() {}
```

Comment:
You don't need to explicitly provide an empty constructor; Java does it for you.